src/components/Accordion/AccordionList.tsx

```diff
@@ -0,0 +1,28 @@
+import type { FC } from "react";
+import type { AccordionProps } from "../../types/AccordionProps";
+import { AccordionItem } from "./AccordionItem";
+
+type Props = {
+  items: AccordionProps[];
+  currentItem: AccordionProps;
+  onCurrentItemClick: (item: AccordionProps) => void;
```

Comment:
```suggestion
  onCurrentItemClick: Dispatch<SetStateAction<AccordionProps>>;
```

Это утилитарный тип для `useState`, который более явно отражает суть функции